src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -219,7 +219,7 @@ void CoilCoolingDXCurveFitPerformance::simulate(EnergyPlus::EnergyPlusData &stat
     this->recoveredEnergyRate = 0.0;
     this->NormalSHR = 0.0;
 
-    if (useAlternateMode == HVAC::coilSubcoolReheatMode) {
+    if (coilMode == HVAC::CoilMode::SubcoolReheat) {
```

Comment:
This should be fine here, but I'm not sure the right spots were changed in https://github.com/NREL/EnergyPlus/pull/10499/commits/a0962524b93fe82ce5ade89005ad1dab2c5e0a86. I'll review shortly.

Reply:
> Why did this compile? should be maxAvailCoilMode? I'm getting confused whether this is max modes or operating mode.

It compiled because this is a method and there is a member named `coilMode` (remember, `this->` is optional). I think you just found the reason for the diff.

This is one of the reasons why you need to have a different naming convention for local variables and member variables.